src/components/AutoDeposit/AutoDepositSignUpForm.vue

```diff
@@ -0,0 +1,475 @@
+					id="donation-checkbox"
+					v-model="donationCheckbox"
+					@change="donationCheckboxChange()"
+				/>
+				<span class="ad-sign-up-form__donation-text">
+					Add an optional
+				</span>
+			</div>
+
+			<div class="columns shrink">
+				<label
+					class="show-for-sr"
+					:class="{ 'error': $v.donation.$invalid }"
+					for="donation"
+				>
+					Donation
+				</label>
+				<kv-dropdown-rounded
+					class="donation-dropdown"
+					v-model="donationOptionSelected"
+					v-if="donationOptionSelected !== 'other'"
+				>
+					<option
+						v-for="(option, index) in dropdownOptions"
+						:value="option.value"
+						:key="index"
+					>
+						{{ option.label }}
+					</option>
+				</kv-dropdown-rounded>
+				<kv-currency-input
+					class="text-input"
+					id="donation"
+					v-model="donation"
+					v-if="donationOptionSelected === 'other'"
+				/>
+			</div>
+
+			<div class="columns shrink">
+				<span class="ad-sign-up-form__donation-text">
+					donation to support Kiva’s costs
+				</span>
+			</div>
+		</div>
+
+		<!-- Donation Errors -->
+		<div class="row column text-center">
+			<ul class="validation-errors" v-if="$v.donation.$invalid">
+				<li v-if="!$v.donation.minValue || !$v.donation.maxValue">
+					Enter a donation amount of $0-$10,000
+				</li>
+			</ul>
+		</div>
+
+		<!-- General Errors & Messaging-->
+		<div class="row column text-center">
+			<ul class="validation-errors"
+				v-if="!$v.adAmount.maxTotal || !$v.donation.maxTotal"
+			>
+				<li>
+					To set up an Auto Deposit, please enter a total amount between $0 and $10,000.
+				</li>
+			</ul>
+			<small v-if="dayOfMonth > 28">
+				(note - deposit may be processed on the last day of the month)</small>
+		</div>
+
+		<!-- Payment Wrapper-->
+		<div class="row column">
+			<div class="payment-dropin-wrapper">
+				<div class="payment-dropin-invalid-cover" v-if="$v.$invalid"></div>
+				<!-- <auto-deposit-drop-in-payment-wrapper
+					:amount="totalCombinedDeposit"
+					:donate-amount="donation"
+					:day-of-month="dayOfMonth"
+					@complete-transaction="completeMGBraintree"
+				/> -->
+			</div>
+			<!-- ! TODO -->
+			<!-- If not logged in, display "Sign in to setup" - button -->
+		</div>
+	</form>
+</template>
+
+<script>
+import numeral from 'numeral';
+import _get from 'lodash/get';
+import gql from 'graphql-tag';
+import { validationMixin } from 'vuelidate';
+import { required, minValue, maxValue } from 'vuelidate/lib/validators';
+
+// import AutoDepositDropInPaymentWrapper from '@/components/AutoDeposit/AutoDepositDropInPaymentWrapper';
+import IconPencil from '@/assets/icons/inline/pencil.svg';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import KvCurrencyInput from '@/components/Kv/KvCurrencyInput';
+import KvDropdownRounded from '@/components/Kv/KvDropdownRounded';
+
+const pageQuery = gql`query monthlyGoodSetupPageControl {
+    general {
+		braintreeDropInFeature: uiConfigSetting(key: "feature.braintree_dropin") {
```

Comment:
I don't think we need this seeing as there wouldn't be any type of fallback...also I wouldn't want it to collide with other drop-in usage elsewhere. We would need to do a series of convoluted steps to be able to redirect back to the old page which I don't think are worth it. Let's just roll with the new new.